.env.factory.mainnet

```diff
@@ -0,0 +1,14 @@
+# L1 token contract address to be bridged to L2, defaults to USDT
```

Comment:
we shouldn't make new file for this, but we should use .env.testnet and .env.mainnet?